src/commands/up.rs

```diff
@@ -165,13 +174,18 @@ impl UpCommand {
 
         let local_app_dir = app_source.local_app_dir().map(Into::into);
 
-        let run_opts = RunTriggerOpts {
-            locked_app,
-            working_dir,
-            local_app_dir,
-        };
-
-        self.run_trigger(trigger_cmd, Some(run_opts)).await
+        join_all(trigger_cmd.iter().map(|cmd| {
```

Comment:
What happens if one of the trigger executors fails to run? If I'm not missing anything I believe `spin up` would happily continue executing the other triggers which probably (?) isn't what we want.

Reply:
I'll look into it.